EIPS/eip-5792.md

```diff
@@ -328,7 +366,11 @@ We also debated whether the methods should be called `wallet_sendTransaction`, `
 
 ### Call Execution Atomicity
 
-The `wallet_sendCalls` method accepts an array of `calls` and by default requires that these calls be executed as part of a single transaction. That said, we expect that in some cases app developers might want to submit batched calls if and only if they can be executed non-atomically. This would enable EOA wallets to accept multiple calls as well over the same interface. A definition of this behaviour may be proposed as a separate capability ERC.
+The `wallet_sendCalls` method accepts an array of `calls`. However, this proposal does not require that these calls be executed as part of a single transaction. It enables EOA wallets to express their ability to execute calls as part of a single transaction (atomically) or as part of multiple transactions (non-atomically). It also enables Apps to express their preference for how calls must be executed (in case a wallet supports both atomic and non-atomic execution).
+
+The `atomicity` special capability that was made an integral part of this specification in order promote expressiveness. And due to its importance to reduce ambiguity between the App and the Wallet, the capability is expressed as a top-level field in the `wallet_sendCalls` request and the `wallet_getCallsStatus` response.
```

Comment:
I think as well as expressiveness, this is to facilitate adoption by EOAs (for example where 7702 isn't available or a user hasn't enabled it for their account)